iCubGenova09/hardware/inertials/left_arm-eb1-IMU.xml

```diff
@@ -38,7 +38,7 @@
             </group>
 
             <group name="SETTINGS">
-                <param name="acquisitionRate">      50      </param>
+                <param name="acquisitionRate">      5      </param>
```

Comment:
Do you have any reference for this kind of change?

Reply:
From https://github.com/robotology/robots-configuration/blob/f4be4b92aba5717dedd13677fe43f7c6b3a3ab5e/iCubTemplates/iCubTemplateV6_0/hardware/inertials/body_part-ebX-inertials.xml#L79-L80

It is `<!-- the acquisition rate of all the sensors in ms. values are in range [5, 200]. If out they are clipped to the nearest limit -->`